File: src/main/java/com/jcfp/tallererp/model/Articulo.java
package com.jcfp.tallererp.model;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

@Entity(name = "articulos")
@Getter
@Setter
public class Articulo{
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String descripcion;
    private double precio;
    private int stock;
    private String proveedor;
}

File: frontend/src/types/entities/articulo.ts
export interface ArticuloPayload {
  descripcion: string;
  precio: string;
  stock: string;
  proveedor: string;
}

export interface ArticuloEditData extends ArticuloPayload {
  id?: string;
}

export interface Articulo extends ArticuloPayload {
  id?: string;
}
